app/services/data_migrations/create_missing_temp_sites.rb

```diff
@@ -0,0 +1,36 @@
+module DataMigrations
+  class CreateMissingTempSites
+    TIMESTAMP = 20220525161544
+    MANUAL_RUN = true
+
+    def change
+      Site
```

Comment:
instead of finding all sites not attached to course options, we would like to find Site attached to course options that have no corresponding temp site instead. Orphaned sites will be removed when we switch to using temp sites